tests/test_ExperimentDefinition.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from ml_experiment.DefinitionPart import DefinitionPart
+from ml_experiment.ExperimentDefinition import ExperimentDefinition
+
+
+def test_ExperimentDefinition():
```

Comment:
This is a great spot for the `tmp_path` fixture, which produces (and cleans up) a temporary file path for storing test results and guaranteeing there is no overlapping global state.
```python
def test_ExperimentDefinition(tmp_path):
  ...
  part = stubbed_DefinitionPart(exp_name, part_name, base=str(tmp_path))
  ...
```

Reply:
Sorcery!